### ISSUE TITLE
The Lesson Plans archive lacks a clear heading, and omits pagination info

### ISSUE THREAD

**Issue by jonoalderson** [WordPress/Learn#96]
For SEO and accessibility reasons, the template at https://learn.wordpress.org/lesson-plans/ should have a clear heading (a `<h1>` tag) which describes the current page. E.g., "Lesson Plans".

In paginated states, the template heading area should also describe the current location (e.g., "Page 4 of 6").

**Comment by jonoalderson** [comment]
Consideration should also be given for how this presents in filtered states (e.g., on https://learn.wordpress.org/lesson-plans/admin/).
